Romuashi/hometask-2/get_middle_charachter.rb

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env ruby
+# frozen_string_literal: true
+
+def get_middle(str)
+  x = str.length / 2
+  str.slice(x - 1, x) if str.length.even?
```

Comment:
you can also use alias of slice method
example: `str[x-1..x]`, it's equal to yours code but looks simpler